shared/requiresSignin.ts

```diff
@@ -5,14 +5,17 @@ const publicPaths = [
 	'/api/contact-us/',
 	'/api/known-issues/',
 	'/api/reminders/cancel/',
+	'/api/reminders/create-public/',
 	'/api/help-centre/',
 	'/cancel-reminders/',
+	'/create-reminder/',
 	'/help-centre/',
 	'/maintenance/',
 ];
 
 export const requiresSignin = (path: string) => {
-	const normalizedPath = normalize(path + '/');
+	const pathWithoutQuerystring = path.split('?')[0];
```

Comment:
Are there any unit tests for `requiresSignin`? It feels like the kind of scenario where a couple of tests might act nicely as documentation (i.e. a path with and without a query string). What do you think?

Reply:
Great point, I've pushed a unit tests file